go.mod

```diff
@@ -1,13 +1,19 @@
 go 1.18
 
+retract [v3.0.0, v3.3.0]
```

Comment:
should we add a comment? Froms go [docs](https://go.dev/ref/mod#go-mod-file-retract)

> Each retract directive should have a comment explaining the rationale for the retraction, though this is not mandatory. The go command may display rationale comments in warnings about retracted versions and in go list output. A rationale comment may be written immediately above a retract directive (without a blank line in between) or afterward on the same line. If a comment appears above a block, it applies to all retract directives within the block that don’t have their own comments. A rationale comment may span multiple lines.

Maybe something like "retracted version depends on SDK version without the dragonberry fix"

Reply:
very yes